src/profile-generator.cc

```diff
@@ -357,7 +395,7 @@ CpuProfile::CpuProfile(const char* title, bool record_samples)
 
 
 void CpuProfile::AddPath(base::TimeTicks timestamp,
-                         const Vector<CodeEntry*>& path, int src_line) {
+                         const Vector<StackEntry>& path, int src_line) {
```

Comment:
I noticed that you store value of the `StackEntry` into the `Vector`. Previously, the CodeEntry is pointer in the `Vector`.  What's the reason to do this change?